README.md

```diff
@@ -95,7 +95,7 @@ If you'd like to contribute to Qiskit Terra, please take a look at our
 [contribution guidelines](CONTRIBUTING.md). This project adheres to Qiskit's [code of conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
 We use [GitHub issues](https://github.com/Qiskit/qiskit-terra/issues) for tracking requests and bugs. Please
-[join the Qiskit Slack community](https://join.slack.com/t/qiskit/shared_invite/zt-e4sscbg2-p8NHTezPVkC3r8nV6BIUVw)
+[join the Qiskit Slack community](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/)
```

Comment:
The footer of qiskit.org uses a url shortener for that. I think it's easier pointing to that instead.

```suggestion
[join the Qiskit Slack community](http://ibm.co/joinqiskitslack)
```